source/dlangbot/app.d

```diff
@@ -393,6 +401,42 @@ void setBotTimer(C)(Duration dur, C callback)
         callback();
 }
 
+// compute the number of points the author will get for fixing any issues
+private uint getTotalPointsPerPR(T)(T bugzillSeverities)
+if (isInputRange!T)
+{
+    import std.algorithm : map, sum;
+    uint severityToPoints(string severity)
+    {
+        switch(severity)
+        {
+            case "enhancement", "trivial":
```

Comment:
I think that it would be easier to simply update the issue severity once a PR was opened for it. PR complexity, number of revisions, age and votes are things that are not easy to mechanically reason about, therefore it is better to have a reviewer decide in such situations.